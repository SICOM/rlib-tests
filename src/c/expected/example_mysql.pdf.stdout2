%PDF-1.3
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [ 6 0 R ]
>>
endobj
2 0 obj
<<
/Type /Font
/Subtype /Type1
/Name /Frpdf0
/BaseFont /Courier
/Encoding /WinAnsiEncoding
>>
endobj
3 0 obj
<<
/Type /Outlines
/Count 0
>>
endobj
4 0 obj
[/PDF /Text ]
endobj
5 0 obj
<</Length 5849>>
stream
1.0000 0.0000 0.0000 1.0000 0.0000 792.0000 cm
0.0000 -1.0000 1.0000 0.0000 0.0 0.0 cm
BT
/Frpdf0 9.000 Tf
1.0000 0.0000 -0.0000 1.0000 14.4000 16.5600 Tm
(Page:   1) Tj
/Frpdf0 12.000 Tf
1.0000 0.0000 -0.0000 1.0000 14.4000 579.4800 Tm
(Products Report) Tj
/Frpdf0 9.000 Tf
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
ET
14.400 561.600 763.200 1.856 re
f
BT
/Frpdf0 11.000 Tf
1.0000 0.0000 -0.0000 1.0000 14.4000 549.2400 Tm
(Products Report \(Page Header\)) Tj
/Frpdf0 9.000 Tf
ET
14.400 541.600 763.200 0.856 re
f
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
14.400 532.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 534.7600 Tm
(Number         ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 534.7600 Tm
( Name                 Type      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 534.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 534.7600 Tm
(Category  ) Tj
ET
14.400 531.600 763.200 0.856 re
f
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 520.7600 Tm
(10000000006    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 520.7600 Tm
( NRS                  9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 520.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 520.7600 Tm
(0         ) Tj
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
ET
14.400 509.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 511.7600 Tm
(10000000008    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 511.7600 Tm
( food sales           9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 511.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 511.7600 Tm
(0         ) Tj
1.0000 0.0000 -0.0000 1.0000 14.4000 502.7600 Tm
(10000000005    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 502.7600 Tm
( overring             9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 502.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 502.7600 Tm
(0         ) Tj
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
ET
14.400 491.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 493.7600 Tm
(10000000007    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 493.7600 Tm
( refund               9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 493.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 493.7600 Tm
(0         ) Tj
1.0000 0.0000 -0.0000 1.0000 14.4000 484.7600 Tm
(10000000010    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 484.7600 Tm
( gift cert sales      9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 484.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 484.7600 Tm
(0         ) Tj
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
ET
14.400 473.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 475.7600 Tm
(10000000011    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 475.7600 Tm
( other sales          9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 475.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 475.7600 Tm
(0         ) Tj
1.0000 0.0000 -0.0000 1.0000 14.4000 466.7600 Tm
(10000000001    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 466.7600 Tm
( EI Sales             9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 466.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 466.7600 Tm
(0         ) Tj
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
ET
14.400 455.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 457.7600 Tm
(10000000002    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 457.7600 Tm
( TO Sales             9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 457.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 457.7600 Tm
(0         ) Tj
1.0000 0.0000 -0.0000 1.0000 14.4000 448.7600 Tm
(10000000003    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 448.7600 Tm
( DT Sales             9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 448.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 448.7600 Tm
(0         ) Tj
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
ET
14.400 437.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 439.7600 Tm
(10000000009    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 439.7600 Tm
( nonfood sales        9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 439.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 439.7600 Tm
(0         ) Tj
1.0000 0.0000 -0.0000 1.0000 14.4000 430.7600 Tm
(10000000013    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 430.7600 Tm
( credits issued       9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 430.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 430.7600 Tm
(0         ) Tj
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
ET
14.400 419.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 421.7600 Tm
(10000000014    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 421.7600 Tm
( promo sales          9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 421.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 421.7600 Tm
(0         ) Tj
1.0000 0.0000 -0.0000 1.0000 14.4000 412.7600 Tm
(10000000015    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 412.7600 Tm
( gift cert redeemed   9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 412.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 412.7600 Tm
(0         ) Tj
0.8980 0.8980 0.8980 rg
0.8980 0.8980 0.8980 RG
ET
14.400 401.600 313.200 8.856 re
f
0.0000 0.0000 0.0000 rg
0.0000 0.0000 0.0000 RG
BT
1.0000 0.0000 -0.0000 1.0000 14.4000 403.7600 Tm
(10000000060    ) Tj
1.0000 0.0000 -0.0000 1.0000 95.4000 403.7600 Tm
( reserved             9999      ) Tj
1.0000 0.0000 -0.0000 1.0000 268.2000 403.7600 Tm
( ) Tj
1.0000 0.0000 -0.0000 1.0000 273.6000 403.7600 Tm
(0         ) Tj
ET

endstream
endobj
6 0 obj
<<
/Type /Page
/Parent 1 0 R
/Resources <<
/Font <<
/Frpdf0 2 0 R
>>
/ProcSet 4 0 R >>
/MediaBox [0 0 612 792]
/CropBox [0 0 612 792]
/Rotate 270
/Contents 5 0 R
>>
endobj
7 0 obj
<<
/Creator (RPDF By SICOM Systems)
/CreationDate (D:19700101000000)
/Producer (RPDF 1.0)
/Author (RPDF)
/Title (RLIB Report)
/Subject (None)
/Keywords (RPDF)
>>
endobj
8 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Outlines 3 0 R
>>
endobj
xref
0 9
0000000000 65535 f 
0000000009 00000 n 
0000000068 00000 n 
0000000177 00000 n 
0000000223 00000 n 
0000000252 00000 n 
0000006151 00000 n 
0000006331 00000 n 
0000006508 00000 n 
trailer
<<
/Size 9
/Root 8 0 R
/Info 7 0 R
>>
startxref
6573
%%EOF
