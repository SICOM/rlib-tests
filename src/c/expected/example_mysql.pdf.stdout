%PDF-1.3
%·¾­ª
1 0 obj
<<
/Type /Catalog
/Pages 2 0 R
>>
endobj
2 0 obj
<<
/Type /Pages
/Kids [ 4 0 R ]
/Count 1
>>
endobj
3 0 obj
<<
/Producer (Haru Free PDF Library 2.2.1)
/Title (RLIB Report)
>>
endobj
4 0 obj
<<
/Type /Page
/MediaBox [ 0 0 612 792 ]
/Contents 5 0 R
/Resources <<
/ProcSet [ /PDF /Text /ImageB /ImageC /ImageI ]
/Font <<
/F1 7 0 R
>>
>>
/Parent 2 0 R
/Rotate 270
>>
endobj
5 0 obj
<<
/Length 6 0 R
>>
stream
1 0 0 1 0 792 cm
0 -1 1 0 0 0 cm
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 16.56 Tm
(Page:   1) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 12 Tf
BT
1 0 0 1 14.4 579.47998 Tm
(Products Report) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
14.4 561.59998 763.20001 1.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 11 Tf
BT
1 0 0 1 14.4 549.23999 Tm
(Products Report \050Page Header\051) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 541.59998 763.20001 0.856 re
f
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 532.59998 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 534.76001 Tm
(Number         ) Tj
ET
BT
1 0 0 1 95.4 534.76001 Tm
( Name                 Type      ) Tj
ET
BT
1 0 0 1 268.20001 534.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 534.76001 Tm
(Category  ) Tj
ET
14.4 531.59998 763.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 520.76001 Tm
(10000000006    ) Tj
ET
BT
1 0 0 1 95.4 520.76001 Tm
( NRS                  9999      ) Tj
ET
BT
1 0 0 1 268.20001 520.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 520.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 509.60001 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 511.76001 Tm
(10000000008    ) Tj
ET
BT
1 0 0 1 95.4 511.76001 Tm
( food sales           9999      ) Tj
ET
BT
1 0 0 1 268.20001 511.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 511.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 502.76001 Tm
(10000000005    ) Tj
ET
BT
1 0 0 1 95.4 502.76001 Tm
( overring             9999      ) Tj
ET
BT
1 0 0 1 268.20001 502.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 502.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 491.60001 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 493.76001 Tm
(10000000007    ) Tj
ET
BT
1 0 0 1 95.4 493.76001 Tm
( refund               9999      ) Tj
ET
BT
1 0 0 1 268.20001 493.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 493.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 484.76001 Tm
(10000000010    ) Tj
ET
BT
1 0 0 1 95.4 484.76001 Tm
( gift cert sales      9999      ) Tj
ET
BT
1 0 0 1 268.20001 484.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 484.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 473.60001 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 475.76001 Tm
(10000000011    ) Tj
ET
BT
1 0 0 1 95.4 475.76001 Tm
( other sales          9999      ) Tj
ET
BT
1 0 0 1 268.20001 475.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 475.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 466.76001 Tm
(10000000001    ) Tj
ET
BT
1 0 0 1 95.4 466.76001 Tm
( EI Sales             9999      ) Tj
ET
BT
1 0 0 1 268.20001 466.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 466.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 455.60001 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 457.76001 Tm
(10000000002    ) Tj
ET
BT
1 0 0 1 95.4 457.76001 Tm
( TO Sales             9999      ) Tj
ET
BT
1 0 0 1 268.20001 457.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 457.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 448.76001 Tm
(10000000003    ) Tj
ET
BT
1 0 0 1 95.4 448.76001 Tm
( DT Sales             9999      ) Tj
ET
BT
1 0 0 1 268.20001 448.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 448.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 437.60001 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 439.76001 Tm
(10000000009    ) Tj
ET
BT
1 0 0 1 95.4 439.76001 Tm
( nonfood sales        9999      ) Tj
ET
BT
1 0 0 1 268.20001 439.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 439.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 430.76001 Tm
(10000000013    ) Tj
ET
BT
1 0 0 1 95.4 430.76001 Tm
( credits issued       9999      ) Tj
ET
BT
1 0 0 1 268.20001 430.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 430.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 419.60001 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 421.76001 Tm
(10000000014    ) Tj
ET
BT
1 0 0 1 95.4 421.76001 Tm
( promo sales          9999      ) Tj
ET
BT
1 0 0 1 268.20001 421.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 421.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 412.76001 Tm
(10000000015    ) Tj
ET
BT
1 0 0 1 95.4 412.76001 Tm
( gift cert redeemed   9999      ) Tj
ET
BT
1 0 0 1 268.20001 412.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 412.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 401.60001 313.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 403.76001 Tm
(10000000060    ) Tj
ET
BT
1 0 0 1 95.4 403.76001 Tm
( reserved             9999      ) Tj
ET
BT
1 0 0 1 268.20001 403.76001 Tm
( ) Tj
ET
BT
1 0 0 1 273.60001 403.76001 Tm
(0         ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf

endstream
endobj
6 0 obj
5421
endobj
7 0 obj
<<
/Type /Font
/BaseFont /Courier
/Subtype /Type1
/Encoding /WinAnsiEncoding
>>
endobj
xref
0 8
0000000000 65535 f
0000000015 00000 n
0000000064 00000 n
0000000123 00000 n
0000000205 00000 n
0000000393 00000 n
0000005868 00000 n
0000005888 00000 n
trailer
<<
/Root 1 0 R
/Info 3 0 R
/Size 8
>>
startxref
5983
%%EOF
